recipes/geant4/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [[ ${HOST} =~ .*darwin.* ]]; then
+  CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
+else
+  CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
+fi
+
+if [[ ${DEBUG_C} == yes ]]; then
+  CMAKE_BUILD_TYPE=Debug
+else
+  CMAKE_BUILD_TYPE=Release
+fi
+
+# cmake_minimum_required(VERSION 3.1) is required to compile
+# the examples to compile using conda's compiler packages
+sed -r -i -E 's#cmake_minimum_required\(VERSION [0-9]\.[0-9]#cmake_minimum_required(VERSION 3.1#gI' \
```

Comment:
@tschoonj So the issue on macOS is that `cmake_minimum_required(VERSION 3.1)` sets a policy in CMake that is required for it to support the conda compilers.

This line patches all the examples so users of the package don't run into the same issue. If you would rather fix it a different way, go ahead.

Reply:
No looks good to me